xbmc/cores/VideoPlayer/Edl.cpp

```diff
@@ -881,141 +869,132 @@ EDL::Action CEdl::GetLastEditActionType() const
+  // Find closest scene backwards
+  else
   {
-    for (int i = 0; i < (int)m_vecSceneMarkers.size(); i++)
+    for (const int& scene : m_sceneMarkers)
     {
-      if ((m_vecSceneMarkers[i] < iSeek) && ((iSeek - m_vecSceneMarkers[i]) < iDiff))
+      if ((scene < seekTime) && ((seekTime - scene) < diff))
       {
-        iDiff = iSeek - m_vecSceneMarkers[i];
-        *iSceneMarker = m_vecSceneMarkers[i];
-        bFound = true;
+        diff = seekTime - scene;
+        *sceneMarker = scene;
+        found = true;
       }
     }
   }
 
-  /*
-   * If the scene marker is in a cut then return the end of the cut. Can't guarantee that this is
-   * picked up when scene markers are added.
-   */
+  // If the scene marker is in a cut then return the end of the cut. Can't guarantee that this is
+  // picked up when scene markers are added.
   Edit edit;
-  if (bFound && InEdit(*iSceneMarker, &edit) && edit.action == Action::CUT)
-    *iSceneMarker = edit.end;
+  if (found && InEdit(*sceneMarker, &edit) && edit.action == Action::CUT)
+    *sceneMarker = edit.end;
 
-  return bFound;
+  return found;
 }
 
-std::string CEdl::MillisecondsToTimeString(const int iMilliseconds)
+std::string CEdl::MillisecondsToTimeString(int msec)
 {
-  std::string strTimeString = StringUtils::SecondsToTimeString((long)(iMilliseconds / 1000), TIME_FORMAT_HH_MM_SS); // milliseconds to seconds
-  strTimeString += StringUtils::Format(".{:03}", iMilliseconds % 1000);
-  return strTimeString;
+  // milliseconds to seconds
+  std::string timeString =
+      StringUtils::SecondsToTimeString(static_cast<long>(msec / 1000), TIME_FORMAT_HH_MM_SS);
+  timeString += StringUtils::Format(".{:03}", msec % 1000);
+  return timeString;
 }
 
 void CEdl::MergeShortCommBreaks()
 {
-  /*
-   * mythcommflag routinely seems to put a 20-40ms commercial break at the start of the recording.
-   *
-   * Remove any spurious short commercial breaks at the very start so they don't interfere with
-   * the algorithms below.
-   */
-  if (!m_vecEdits.empty() && m_vecEdits[0].action == Action::COMM_BREAK &&
-      (m_vecEdits[0].end - m_vecEdits[0].start) < 5 * 1000) // 5 seconds
+  // mythcommflag routinely seems to put a 20-40ms commercial break at the start of the recording.
+  // Remove any spurious short commercial breaks at the very start so they don't interfere with
+  // the algorithms below.
+  if (!m_edits.empty() && m_edits.front().action == Action::COMM_BREAK &&
+      (m_edits.front().end - m_edits.front().start) < 5 * 1000) // 5 seconds
   {
-    CLog::Log(LOGDEBUG, "{} - Removing short commercial break at start [{} - {}]. <5 seconds",
-              __FUNCTION__, MillisecondsToTimeString(m_vecEdits[0].start),
-              MillisecondsToTimeString(m_vecEdits[0].end));
-    m_vecEdits.erase(m_vecEdits.begin());
+    CLog::LogF(LOGDEBUG, "Removing short commercial break at start [{} - {}]. <5 seconds",
+               MillisecondsToTimeString(m_edits[0].start),
```

Comment:
Some places use the index operator and some use `.front()` it may be nice to choose one for consistency